### PR TITLE
CP-17652: Add PVS_proxy feature flag

### PR DESCRIPTION
**Pull request by johnelse**

### CODE REVIEW

**Comment by lindig** [comment]
That looks good.

File: ocaml/xapi/features.ml [hunk 1]
```diff
@@ -87,6 +88,7 @@ let keys_of_features =
 		Xen_motion, ("restrict_xen_motion", Negative, "XenMotion");
 		Guest_ip_setting, ("restrict_guest_ip_setting", Negative, "GuestIP");
 		AD, ("restrict_ad", Negative, "AD");
+		PVS_proxy, ("restrict_pvs_cache", Negative, "PVS_proxy");
```
Comment (johnelse) [inline ocaml/xapi/features.ml]: Oops, should be `restrict_pvs_proxy`

**Comment by johnelse** [comment]
I'm going to close this for now, and make a new one with the rest of the code changes.

**Comment by johnelse** [comment]
This is ready for review now.

**Comment by robhoes** [comment]
:shipit:

**Pull request merged**